core/src/main/scala/chisel3/Data.scala

```diff
@@ -157,6 +157,26 @@ package experimental {
       target.direction
     }
 
+    private def isBinding[B <: ConstrainedBinding : Manifest](target: Data) = {
+      target.topBindingOpt match {
+        case Some(b: B) => true
+        case _ => false
+      }
```

Comment:
```suggestion
    private def hasBinding[B <: ConstrainedBinding : ClassTag](target: Data) = {
      target.topBindingOpt match {
        case Some(b: B) => true
        case _ => false
      }
```
`hasBinding` is more grammatically correct.
`Manifest` is old and seems to be discouraged by the docs, use `scala.reflect.ClassTag` instead.